server/src/utils/mixerConnections/SSLMixerConnection.ts

```diff
@@ -226,47 +220,9 @@ export class SSLMixerConnection {
                     ) {
                         lastWasAck = false
                         // MUTE ON/OFF COMMAND
-                        let commandHex = buffer.toString('hex')
-                        let channelIndex = buffer[6]
-                        let value: boolean = buffer[7] === 0 ? true : false
-                        logger.trace(
-                            `Receive Buffer Channel On/off: ${this.formatHexWithSpaces(
-                                commandHex,
-                                ' ',
-                                2
-                            )}`
-                        )
-
-                        let assignedFaderIndex =
-                            state.channels[0].chMixerConnection[this.mixerIndex]
-                                .channel[channelIndex].assignedFader
-
-                        store.dispatch(storeSetMute(assignedFaderIndex, value))
-
-                        if (remoteConnections) {
-                            remoteConnections.updateRemoteFaderState(
-                                assignedFaderIndex,
-                                value ? 1 : 0
-                            )
-                        }
-                        state.channels[0].chMixerConnection[
-                            this.mixerIndex
-                        ].channel.forEach((channel: any, index: number) => {
-                            if (
-                                channel.assignedFader === assignedFaderIndex &&
-                                index !== channelIndex
-                            ) {
-                                this.updateMuteState(
-                                    index,
-                                    state.faders[0].fader[assignedFaderIndex]
-                                        .muteOn
-                                )
-                            }
-                        })
-                        global.mainThreadHandler.updatePartialStore(
-                            assignedFaderIndex
-                        )
+                        this.handleReceivedMuteCommand(buffer)
                     } else {
+                        // UNKNOWN COMMAND

```

Comment:
Wouldn't it be better to log that we encountered an unknown command (depending on the volume encountered), than having a comment stating this?